terraform/vars/adoagent.tfvars

```diff
@@ -22,4 +22,18 @@ nsg_name = "vh-infra-core-ado-nsg"
 # environment
 env = "stg"
 
-dns_zone = ["dev.platform.hmcts.net", "test.platform.hmcts.net", "ithc.platform.hmcts.net", "staging.platform.hmcts.net", "platform.hmcts.net", "demo.platform.hmcts.net"]
\ No newline at end of file
+dns_zone = ["dev.platform.hmcts.net", "test.platform.hmcts.net", "ithc.platform.hmcts.net", "staging.platform.hmcts.net", "platform.hmcts.net", "demo.platform.hmcts.net"]
```

Comment:
missing the `sandbox` DNS and also the `hearings.reform.hmcts.net`

Reply:
these do not exist